transport-native-unix-common/src/main/java/io/netty/channel/unix/Limits.java

```diff
@@ -21,11 +21,22 @@
 import static io.netty.channel.unix.LimitsStaticallyReferencedJniMethods.uioMaxIov;
 
 public final class Limits {
-    public static final int IOV_MAX = iovMax();
```

Comment:
> and it looks like same efficiency as before.

Unless you can demonstrate performance isn't impacted please revert this portion of the change. A potential alternative maybe to introduce an additional method we can use in tests:

```java
public static boolean isBufferCopyNeededForWrite(ByteBuf byteBuf) {
  return isBufferCopyNeededForWrite(byteBuf, IOV_MAX);
}

static boolean isBufferCopyNeededForWrite(ByteBuf byteBuf, int iovMax) {
 return ...;
}
```